### ISSUE TITLE
[Desktop Bug] Color Picker Previews in Style Settings Not Holding Colors

### ISSUE THREAD

**Issue by SuperDragonite2172** [ArtexJay/Obsidian-CyberGlow#72]
**Describe the bug**
When editing the colors in the Color Picker dropdown in Style Settings, the color previews don't hold their color after closing and re-opening Style Settings.  They all default to the `hsla(170, 49%, 32%, 1)` regardless of what they're actually set to.

**To Reproduce**
1. Open Style Settings in Options
2. Change a color or apply the default
3. Close the dropdown or options
4. Observe the new colors taking effect
5. Reopen the settings to find the colors preview "unchanged".

**Expected behavior**
Color previews in Style Settings display the actual color not the default.

**Screenshots, Gifs, Video**
![2024-02-18_1708302823](https://github.com/ArtexJay/Obsidian-CyberGlow/assets/14007732/acd86c73-917b-4343-bfe0-9c7d21464d7d)
![2024-02-18_1708302885](https://github.com/ArtexJay/Obsidian-CyberGlow/assets/14007732/b413ad4b-df6c-4dee-bfa5-1d02ad19fafd)

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro
 - Obsidian Version 1.5.3
 - CyberGlow Version 8.2.5

**Additional context**
The example screenshots show the headings specifically since they're the most obvious, but it applies to all the colors.
Originally I thought the colors weren't applying correctly at all but in filing the issue I discovered it was just Style Settings not updating correctly.  Apologies if there's any weirdness in the information provided as a result.

**Comment by ArtexJay** [comment]
I am unable to reproduce this issue with default theme colors after I applied a custom colors via Style Settings, I have closed style settings and even restarted Obsidian, with Style Setting still showing the correct saved color for headings when altered.
Do they default to that `hsla(170, 49%, 32%, 1)` even with the default theme colors for you?

If you have modified my theme, backup the theme file so you don't lose your custom edits and redownload my theme, see if the issue persists.

**Comment by SuperDragonite2172** [comment]
I uninstalled Cyber Glow and Style Settings, then re-installed both.  I also updated Cyber Glow to 8.2.6.  The issue persists and the displayed colors are still not correct until I reset them or click on the color to change it.  Thankfully the only settings I really change are enabling the workspace background and hovering the status bar.

**Comment by ArtexJay** [comment]
that's very strange, I tried again today, but I cannot replicate this on my end, the colors display correct custom applied colors.
![image](https://github.com/ArtexJay/Obsidian-CyberGlow/assets/32932497/637a9abd-af66-4800-ae77-0b66d6e67c12)

This could be a Style Settings issue, but I really do not know.

My installer version is 1.5.3, with Obsidian at version 1.5.8, I'm assuming it's the same with you.

**Comment by SuperDragonite2172** [comment]
I was on Obsidian 1.5.3.  I just updated it to 1.5.8 and the issue still persists.

**Comment by ArtexJay** [comment]
Are you still seeing this issue in the latest version of CyberGlow v1.4.0?

**Comment by SuperDragonite2172** [comment]
Yes, I'm still seeing the issue.  I'm on Obsidian version 1.7.4, CyberGlow version 1.4.0, and Style Settings 1.0.9.  The colors swatches still all show up as `hsl(170, 49%, 32%)` when I first open the settings and then correct themselves when I click on the swatch.  See the screenshot below for what it looks like immediately after opening the settings.

![First Appearance](https://github.com/user-attachments/assets/9c89960c-8e18-41e2-b668-82895a786583)

**Comment by ThePharaohArt** [comment]
I am still unable to replicate this issue unfortunately; the theme does not interact with the "--pcr-color" element at all which is in charge of the color picker boxes.

With version 2.0.0 do they still display as hsl(170, 49%, 32%) or is it now the new accent color?

**Comment by SuperDragonite2172** [comment]
The behavior has changed slightly.  As of at least 2.0.9, All the color picker boxes default to chosen "Header 6 Color".  If I change the color, close and reopen the Style Settings page, it reflects that new color.  If there's any other sort of information I can provide to aid debugging, let me know.
<img width="723" height="454" alt="Image" src="https://github.com/user-attachments/assets/cb136613-c333-43ba-99b7-19f952887fcd" />

**Comment by ThePharaohArt** [comment]
> The behavior has changed slightly. As of at least 2.0.9, 

this was just released a few minutes ago, did you have the same behavior in 2.0.5?

**Comment by SuperDragonite2172** [comment]
Yes, the behavior was similar.  I didn't keep track of when it changed behavior but my guess is it happened with the 2.0 series.

**Comment by ThePharaohArt** [comment]
try to add to the css code, this is really feeling like a Style Setting bug to me

```
.style-settings-container {
  --pcr-color: none !important;
}
```

**Comment by ThePharaohArt** [comment]
what seems to be happening is for some reason style settings applies the last most color as a color for the entire container, which is causing the issue, in my case it only flashes that color but for you it sounds like it remains that color.

**Comment by ThePharaohArt** [comment]
> put that snippet at the bottom of the CSS file and now they all show up as if the alpha was set to 0.

even when you put in a custom color?

**Comment by SuperDragonite2172** [comment]
Yes, even when I put in a custom color.
